plugins/modules/ipaautomountlocation.py

```diff
@@ -78,15 +78,14 @@ class AutomountLocation(FreeIPABaseModule):
     ipa_param_mapping = {}
 
     def get_location(self, location):
-        response = dict()
         try:
-            response = self.api_command("automountlocation_show",
-                                        location,
-                                        {})
+            response = self.api_command(
+                "automountlocation_show", location, {"all": True}
+            )
         except ipalib_errors.NotFound:
-            pass
-
-        return response.get("result", None)
+            return None
+        else:
```

Comment:
Is this else required by the new pylint?

Reply:
The method modification is what makes most sense here.

It makes it more similar to other `find` methods and helps fixing one warning on the new Pylint (about the use of `dict()`).